website/sagexit/urls.py

```diff
@@ -1,10 +1,18 @@
-from django.contrib import admin
+from django.contrib import admin, auth
+from django.shortcuts import redirect
 from django.urls import path, include
 from django.views.generic import RedirectView
 
+
+def logout(request):
+    auth.logout(request)
+    return redirect("/")
+
+
 urlpatterns = [
     path("", RedirectView.as_view(url="/reservations")),
     path("reservations/", include("room_reservation.urls")),
     path("admin/", admin.site.urls),
+    path("logout/", logout, name="logout"),
```

Comment:
Can you not just do
```suggestion
    path("logout/", LogoutView.as_view(), name="logout"),
```
from `django.contrib.auth.views` -> https://github.com/django/django/blob/main/django/contrib/auth/views.py?

Reply:
If you set the LOGOUT_REDIRECT_URL setting to / it will even redirect for you

Reply:
Hmm you're right. I tried it first with the admin logout view and that didn't work, but this should have worked